### ISSUE TITLE
Quickpanel to navigate bibliography

### ISSUE THREAD

**Issue by jdumas** [SublimeText/LaTeXTools#1157]
Hi,

I like the quickpanel that shows up when we enter a `\cite{}` command in a latex document. I was wondering if it would be possible to trigger it from anywhere to navigate through bibtex entries in a project? E.g. when I have multiple `.bib` file, it would allow me to find an entry based on a fuzzy match with the paper title. Even within a single bib file, it could help navigate the different entries it contains (but I think it is important that we keep the ability to jump to an entry based on their bibtex key).

**Comment by r-stein** [comment]
So you want to have something like C-r for the bibentries?

**Comment by jdumas** [comment]
Yes, pretty much! =) (the panel that appears when we type `\cite{}` is already fine I think).

**Comment by r-stein** [comment]
The problem is that we don't have the file name and position from bibliography parser, but it will be trivial if we have that. I will have a look at that.

**Comment by jdumas** [comment]
I think it might pose a problem if there is a multi-root project, with several latex documents that can include different `.bib` files. How about creating a dummy latex document with *all* the `.bib` files present in the current project? I would suggest 3 different behaviors:
- When called from within a `.bib` file, search only the entries of this particular file.
- When called from a `.tex` file, search only the entries of the current latex document (the context is the same as when we would type a `\cite{}` command).
- When called from elsewhere, use the info from a dummy latex document that include all the `.bib` files in the current project hierarchy.

Let me know if that makes sense.

**Comment by r-stein** [comment]
1 und 2 makes sense and I would do it the same, but calling from somewhere else doesn't make sense to me =)

**Comment by jdumas** [comment]
My use case would be to navigate my article library, without the need to open a latex document. E.g. if I have one folder per theme, each containing its own `.bib` file properly written, jumping to an article from anywhere can prove useful (I don't have to open a latex file first). As another use case, imagine I am writing documentation for my code and want to reference a particular paper. It would be useful to quickly jump to it and be able to cc the reference (e.g. bibkey in doxygen, etc.). Sure, it goes a bit beyond the scope of LaTeXTools (if we are talking about doxygen documentation), but I think it would be a small addition to have.

**Comment by deathaxe** [comment]
ST4 indexes bib files and exposes bibliography keys in global symbol list, reachable via ctrl+shift+r. It doesn't contain rich information as cite completions, but provides basic support for OP's request.
